src/player.cpp

```diff
@@ -5353,7 +5357,7 @@ void player::use( item_location loc )
                                      used.is_medication() ||
                                      used.get_contained().is_food() ||
                                      used.get_contained().is_medication() ) ) {
-        consume( inventory_position );
```

Comment:
Removing this invocation made inventory_location unused, it's generating warnings now.